incubator/hnc/pkg/controllers/suite_test.go

```diff
@@ -41,12 +42,17 @@ import (
 // http://onsi.github.io/ginkgo/ to learn more about Ginkgo.
 
 var (
-	cfg        *rest.Config
-	k8sClient  client.Client
-	testEnv    *envtest.Environment
-	k8sManager ctrl.Manager
+	cfg                 *rest.Config
+	k8sClient           client.Client
+	testEnv             *envtest.Environment
+	k8sManager          ctrl.Manager
+	newObjectController bool
 )
 
+func init() {
+	flag.BoolVar(&newObjectController, "enable-new-object-controller", false, "Enables new object controller.")
```

Comment:
Can you also add something to the makefile so I can say:

```
NOC=1 make test
```

or something like that? ("NOC" = "New Object Controller")